File: Swift-C++-Testsuite/test/arithmetic-operators/Addition/Swift/UnaryPlus.swift
@main
struct MyMain {

    static func main() {
        let _ = unaryPlus(x: -1)
    }

    // Documentation states the following
    // The unary plus operator (+) simply returns the value it operates on, without any change
    static func unaryPlus(x: Int) -> Int {
        var a = x
        var b = +a
        return b
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-2/UnaryPlus.cpp

#include <iostream>

struct MyMain {

    static void main() {
        std::cout << unaryPlus(x: -1);
    }

    // Documentation states the following
    // The unary plus operator (+) simply returns the value it operates on, without any change
    static int unaryPlus(int x) {
        int a = x;
        int b = +a;
        return b;
    }
};

int main() {
    MyMain::main();
    return 0;
}